### DISCUSSION TITLE
Connect blockchain to Magma core

### DISCUSSION THREAD

**Discussion by MerouaM** [magma/magma#14891] [Q&A]
Hello everyone,

I've seen on the Magma core website that Pollen networks are using Magma. My question is about how to connect Magma core to a blockchain (e.g., Ethereum blockchain) on network level ?

Thank you.

Best regards,

**Comment by lucasgonze** [comment]
@MerouaM: this is too broad a question to answer well. Can you narrow down what you are asking?

**Comment by MerouaM** [comment]
Thank you for your answer.

My question is about how to connect an Ethereum blockchain to Magma core via the Federation Gateway? Is it even possible to do this ? if so, do you have any idea about how to do it ?

Best regards,

**Reply by lucasgonze** [reply]
The Ethereum component would fit as a subscriber management feature. If I were you I would carefully read the architectural documentation:

https://magma.github.io/magma/docs/lte/architecture_overview

**Comment by bensternthal** [comment]
Closing discussion, as we have not heard back from the original poster in quite awhile. Feel free to reopen as needed.
